Kryptonite/SessionManager.swift

```diff
@@ -101,8 +103,12 @@ class SessionManager {
         defer { mutex.unlock() }
         mutex.lock()
 
-        let _ = KeychainStorage().delete(key: Session.KeychainKey.pub.tag(for: session.id))
-        let _ = KeychainStorage().delete(key: Session.KeychainKey.priv.tag(for: session.id))
+        do {
+            try KeychainStorage().delete(key: Session.KeychainKey.pub.tag(for: session.id))
```

Comment:
should we separate these into 2 separate `do` blocks so failure of the first doesn't prevent the second?